### ISSUE TITLE
[5.2.0-dev] PHP Fatal error:  Allowed memory size of X bytes exhausted

### ISSUE THREAD

**Issue by PhilETaylor** [symfony/symfony#39056]
**Symfony version(s) affected**: 5.2.0-RC1. 5.2.0-dev

**Description**  
As soon as you install `composer require knplabs/doctrine-behaviors` into a fresh install of Symfony 5.2.0-RC1 (or "next") you get PHP exhausted memory errors

I have tested this, and reproduced it, with a 8Gb `memory_limit`  in `PHP 7.4.12` and `PHP 8.0.0-dev`.

**How to reproduce**  

Simple :) 

```
mkdir my_project && cd my_project
symfony new  --full . --version=next
composer require knplabs/doctrine-behaviors
```

**Possible Solution**  
Sorry, I don't have a solution, I can only report the issue this time. 

**Additional context**  
@jderusse [mentioned here](https://github.com/symfony/symfony/pull/39043#issuecomment-725272995) that #39015 might be related, but I manually reverted that PR and the problem persisted for me. 

**screenshot of whole reproducer**

<img width="961" alt="Screenshot 2020-11-11 at 15 07 51" src="https://user-images.githubusercontent.com/400092/98828215-b72a5600-242f-11eb-9cde-2dded23cfe57.png">

**Comment by Jean85** [comment]
Does https://github.com/symfony/symfony/compare/85d8004602f0b4e18999ade5cb8ecabfa6a06f42..d3b951b767e70a40c80ddbb9ebfcfe7518f14c81 fix it for you too?

**Comment by PhilETaylor** [comment]
As stated in the issue description - no. 

> @jderusse mentioned here that #39015 might be related, but I manually reverted that PR and the problem persisted for me.

To be clear, I get the memory error before and after the changes made in the commit. The change has no effect on fixing the problem.

**Comment by PhilETaylor** [comment]
Closing as appears fixed in #39058 thanks @jderusse !
